pkg/version/version.go

```diff
@@ -0,0 +1,35 @@
+package version
+
+import (
+	"k8s.io/apimachinery/pkg/version"
+	// go mod won't pull in code that isn't depended upon, but we have some code we don't depend on from code that
+	// must be included for our build to work.
+	_ "github.com/openshift/build-machinery-go/make/lib"
+)
+
+var (
+	// commitFromGit is a constant representing the source version that
+	// generated this build. It should be set during build via -ldflags.
+	commitFromGit string
+	// versionFromGit is a constant representing the version tag that
+	// generated this build. It should be set during build via -ldflags.
+	versionFromGit string
+	// major version
+	majorFromGit string
+	// minor version
+	minorFromGit string
+	// build date in ISO8601 format, output of $(date -u +'%Y-%m-%dT%H:%M:%SZ')
+	buildDate string
+)
+
+// Get returns the overall codebase version. It's for detecting
+// what code a binary was built from.
+func Get() version.Info {
```

Comment:
We don't need any of this, the repo is built with go 1.18. Please get this infro from runtime/debug.ReadBuildInfo (The `Settings` should include git information)

Reply:
Do you mean we use the commit id, which could be gotten from runtime/debug.ReadBuildInfo.Settings[vcs.revision], as the version value, right?

Reply:
Sure you can, but not sure if needed. The go version is configured here: https://github.com/openshift/hypershift/blob/main/.ci-operator.yaml#L4